modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -160,15 +170,69 @@ spec:
   baseDomain: example.openshift.com
   privateZone: <1>
     id: mycluster-100419-private-zone
+ifndef::user-infra-vpc[]
   publicZone: <1>
     id: example.openshift.com
+endif::user-infra-vpc[]
 status: {}
 ----
-<1> Remove these sections completely.
+<1> Remove this section completely.
 +
+ifndef::user-infra-vpc[]
 If you do so, you must add ingress DNS records manually in a later step.
+endif::user-infra-vpc[]
 endif::gcp,aws,azure[]
 
+ifdef::user-infra-vpc[]
+. Configure the cloud provider for your VPC.
++
+--
+.. Open the `<installation_directory>/manifests/cloud-provider-config.yaml` file.
+.. Add the `network-project-id` parameter and set its value to the name of project that hoststhe shared VPC.
+.. Add the `network-name` parameter and set its value to the name of the network that hosts the shared VPC.
+.. Replace the value of the `subnetwork-name` parameter with the value of the subnet on the shared VPC to host your compute machines.
```

Comment:
I suggest to change it to:
Replace the value of the subnetwork-name parameter with the value of the shared VPC subnet that hosts your compute machines.